babel.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  plugins: ['lodash', 'angularjs-annotate', ['i18next-extract', { outputPath: 'translations/{{locale}}/{{ns}}.json' }]],
+  plugins: ['lodash', 'angularjs-annotate'],
```

Comment:
I've removed the extraction tool, as it's outdated and buggy. Since we don't support i18n across Portainer, it's still not important to extract keys